docs/docusaurus.config.js

```diff
@@ -57,23 +57,20 @@ const config = {
         redirects: [
           {
             to: '/getting-started',
-            from: ['/tutorial/getting-started','/tutorial/getting-started-http'],
+            from: ['/tutorial/getting-started', '/tutorial/getting-started-http'],
```

Comment:
@lukegalbraithrussell 👋🏼  When you make docs changes, make sure to run the linter so these aren't picked up by subsequent PRs. It'll help us keep the changes for each PR scoped to what they're actually about.

Reply:
hmmm i think maybe i ran the wrong linter because it changed all of this just in my last PR. (Most notably in that sidebar.js file -- I would never manually use double quotations willingly so i remember it changing them all). I'll figure out what happened